ctapipe/reco/hillas_intersection.py

```diff
@@ -220,7 +221,7 @@ def reconstruct_tilted(self, hillas_parameters, tel_x, tel_y,
 
         tx = np.zeros((len(tel_x), 2))
         ty = np.zeros((len(tel_y), 2))
-        for i in range(len(tel_x)):
+        for i in enumerate(tel_x):
```

Comment:
I don't think this is correct.  Should be something like:

```py3
for i, _ in enumerate(tel_x):
    ...
```
since enumerate returns the index and the value as a tuple